src/univariate/continuous/johnson.jl

```diff
@@ -0,0 +1,141 @@
+"""
+    Johnson(ξ, λ, γ, δ)
+
+The Johnson's ``S_U``-distribution with parameters ξ, λ, γ and δ is a transformation of the normal distribution:
+
+```math
+z \\sim \\gamma + \\delta \\sinh^{-1}\\Big(\\frac{x-\\xi}{\\lambda}\\Big),
+```
+
+where ``z \\sim \\mathcal{N}(0,1)``.
```

Comment:
It seems this is copied from Wikipedia (apart from a `\sim` instead of a `=` in the first equation which should be changed) but IMO this needs more explanations. For instance, you could add something like: if `X` is defined according to .... where `Z ~ N(0, 1)`, then `X ~ Johnson(....)`.

Reply:
Thanks for pointing out the `\sim` typo. I'll fix that.

I'll also add

```
i.e. if a random variable ``X`` is defined to be

```math
X = \\lambda\\sinh\\Bigg( \\frac{Z - \\gamma}{\\delta} \\Bigg) + \\xi,
```.

where ``Z \\sim \\mathcal{N}(0,1)``,  then ``X \\sim {\\rm Johnson}(\\xi, \\lambda, \\gamma, \\delta)``.
```

The dot after the enclosing math code block fence is only added here to avoid GitHub from closing the code block there.